### ISSUE TITLE
[Feature request] A11y @vuepress/plugin-slimsearch

### ISSUE THREAD

**Issue by BigSinner** [vuepress/ecosystem#411]
### Clear and concise description of the problem

The result of the search is not accssible. 
Neither the button to reset the search nor the result list have corresponding labels.

### Suggested solution

At least the aria-labels should be set.

**Comment by Mister-Hope** [comment]
https://github.com/vuepress/ecosystem/commit/a34a44b1c7693f55bb9667059adc5eb579f72796
